ironfish/src/primitives/mintData.ts

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+export interface MintData {
+  assetId?: Buffer
+  name: string
+  metadata: string
+  value: bigint
+  isNewAsset: boolean
+}
```

Comment:
this is not a primitive, this is just the interface for a mint in the wallet we should inline this or move it into the wallet.